openpype/hosts/max/plugins/publish/validate_renderpasses.py

```diff
@@ -0,0 +1,159 @@
+                rt.rendOutputFilename).replace(
+                    "\\", "/").split("/")[-1]
+            invalid.append(("Invalid Render Output Folder",
+                            invalid_folder_name))
+        beauty_fname = os.path.basename(rt.rendOutputFilename)
+        ext = os.path.splitext(beauty_fname)[-1].lstrip(".")
+        invalid_image_format = cls.get_invalid_image_format(instance, ext)
+        invalid.extend(invalid_image_format)
+        renderer = instance.data["renderer"]
+        if renderer in [
+            "ART_Renderer",
+            "Redshift_Renderer",
+            "V_Ray_6_Hotfix_3",
+            "V_Ray_GPU_6_Hotfix_3",
+            "Default_Scanline_Renderer",
+            "Quicksilver_Hardware_Renderer",
+        ]:
+            render_elem = rt.maxOps.GetCurRenderElementMgr()
+            render_elem_num = render_elem.NumRenderElements()
+            for i in range(render_elem_num):
+                renderlayer_name = render_elem.GetRenderElement(i)
+                renderpass = str(renderlayer_name).split(":")[-1]
+                rend_file = render_elem.GetRenderElementFilename(i)
+                if not rend_file:
+                    cls.log.error(
+                        f"No filepath for render element {renderpass}")
+                    invalid.append((f"Invalid {renderpass}",
+                                    "No filepath"))
+                rend_fname, ext = os.path.splitext(
+                    os.path.basename(rend_file))
+                if not rend_fname.lstrip(".").endswith(renderpass):
+                    cls.log.error(
+                        f"Filename for {renderpass} should "
+                        f"end with {renderpass}"
+                    )
+                    invalid.append((f"Invalid {renderpass}",
+                                    os.path.basename(rend_file)))
+                invalid_image_format = cls.get_invalid_image_format(
+                    instance, ext)
+                invalid.extend(invalid_image_format)
+        elif renderer == "Arnold":
+            cls.log.debug(
+                "Renderpass validation not supported Arnold yet,"
+                " validation skipped...")
+
+        return invalid
+
+    @classmethod
+    def get_invalid_image_format(cls, instance, ext):
+        """Function to check if the image format of the render outputs
+        aligns with that in the setting.
+
+        Args:
+            instance (pyblish.api.Instance): instance
+            ext (str): image extension
+
+        Returns:
+            list: list of files with invalid image format
+        """
+        invalid = []
+        settings = instance.context.data["project_settings"].get("max")
+        image_format = settings["RenderSettings"]["image_format"]
+        ext = ext.lstrip(".")
+        if ext != image_format:
+            msg = (
+                f"Invalid image format {ext} for render outputs"
+                f"Should be : {image_format}")
```

Comment:
Missing line break and dot?
```suggestion
                f"Invalid image format {ext} for render outputs.\n"
                f"Should be: {image_format}")
```